IntroToCSharp/Pages/Lessons/Variables/TheTypeSystem.razor

```diff
@@ -0,0 +1,141 @@
+        char data type is a special type of integer for which each integer value
+        represents a symbol to be displayed. The char data type uses 2 bytes of
+        data which means it has 65,536 distinct characters.
+    </MudText>
+    <Note>The characters that can be displayed with the </Note>
+
+    <MudText>
+        Unlike a string, a char is created by placing the character inside of
+        single quotes like this:
+    </MudText>
+    <CodeBlock AllowCopy>char someLetter = 'j';</CodeBlock>
+    <Note>You must place <b>exactly</b> on character between the single quotes.</Note>
+    <MudText>
+        Another interesting thing to note is that a char is not limited to the keys
+        on the keyboard. You can place any <Keyword Word="unicode"/> character between
+        the single quotes.
+    </MudText>
+    <CodeBlock AllowCopy ReplIt="JosephCollard/BallChar">
+char ball = '⚽';
+Console.WriteLine(ball);
+    </CodeBlock>
+    <MudText>
+        Each character has a hexidecimal <b>unicode</b> encoding associated with
+        it. If you happen to know the value (or search for one using your
+        favorite search engine), you can enter it using an <Keyword Word="escape
+        code"/>:
+    </MudText>
+    <CodeBlock AllowCopy ReplIt="JosephCollard/SmileyChar">
+char smiley = '\u263A';
+Console.WriteLine(smiley);
+</CodeBlock>
+    <Note>
+        When you use an escape code like the one above, you are only providing a
+        single character between the single quotes.
+    </Note>
+</Section>
+
+<Section Title="Floating-Point Numbers">
+    <MudText>
+        Integers are great for whole numbers, but what happens when we want to
+        store decimals or even repeating decimals like π? Floating-point numbers
+        are a data type that allows programmers to <b>approximate</b> a real number.
+    </MudText>
+    <MudElement Class="ma-4">
+        <DataTable Headers=@floatHeaders Rows=@floatRows/>
+    </MudElement>
+    <MudText>
+        Be careful when using the float and double types. As stated above, these
+        two types are not precise values. That is, they are only approximations that
+        should be "close-enough" to a value. If you need a decimal number to
+        have exact precision, you should use the decimal type. To see an example
+        of a rounding error, run the example below:
+    </MudText>
+
+    <CodeBlock AllowCopy ReplIt="JosephCollard/Floating-Point-Errors">
+float flt = 2.0F - 1.9F;
+double dbl = 2.0 - 1.9;
+decimal dec = 2.0M - 1.9M;
+Console.WriteLine($"Float: {flt}");
+Console.WriteLine($"Double: {dbl}");
+Console.WriteLine($"Decimal: {dec}");
+    </CodeBlock>
+    <MudText>
+        In the above example, you might expect the result of all three values to
+        be <Code>0.1</Code>. However, because float and double are
+        approximations, we see a <Keyword Word="rounding error"/> in the output.
+    </MudText>
+</Section>
+
+<Section Title="Booleans">
```

Comment:
Not sure if you want to add that booleans are either 0 or 1 as well.

Reply:
In C# this is not the case.